app/controllers/application_controller.rb

```diff
@@ -12,8 +12,10 @@ def check_logged_in
     end
   end
 
-  def check_edit_authority
-    flash[:alert] = '自分以外のユーザーは編集・削除できません'
-    redirect_to users_path unless current_user&.id == params[:id].to_i
+  def check_edit_authority(user_id)
+    unless current_user&.id == user_id
+      flash[:alert] = '自分以外のユーザーは編集・削除できません'
+      redirect_to users_path
+    end
```

Comment:
Idiomaticの観点

早期リターンを使おう

```suggestion
  def check_edit_authority(user_id)
    return if current_user&.id == user_id
    
    flash[:alert] = '自分以外のユーザーは編集・削除できません'
    redirect_to users_path
```

Reply:
if修飾子の書き方に慣れない...
下記コミットで修正しました
https://github.com/ok-os-job-change-team/kaito-twitter-clone-bootcamp/pull/46/commits/c999712edbdb8d7138b1f3ab07c346c22874e152